### PR TITLE
lice: 0.4 -> 0.6

### LINKED ISSUE #75521: lice: exception on startup
**Describe the bug**

`lice` fails to run, doesn't find `pkg_resources`.

**To Reproduce**

```
> nix run nixpkgs.lice -c lice
[1 copied (0.3 MiB), 0.0 MiB DL]
Traceback (most recent call last):
  File "/nix/store/jrvln64pbjbwn9cbw13fzjrmckhnckrb-python3.7-lice-0.4/bin/.lice-wrapped", line 11, in <module>
    sys.exit(main())
  File "/nix/store/jrvln64pbjbwn9cbw13fzjrmckhnckrb-python3.7-lice-0.4/lib/python3.7/site-packages/lice/__init__.py", line 5, in main
    from lice.core import main
  File "/nix/store/jrvln64pbjbwn9cbw13fzjrmckhnckrb-python3.7-lice-0.4/lib/python3.7/site-packages/lice/core.py", line 1, in <module>
    from pkg_resources import (resource_stream, resource_listdir)
ModuleNotFoundError: No module named 'pkg_resources'
```

**Metadata**
Please run `nix run nixpkgs.nix-info -c nix-info -m` and paste the result.

 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.88, NixOS, 19.09.1549.45ea6092203 (Loris)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3`
 - channels(root): `"home-manager, nixos-19.09.1549.45ea6092203, nixos-unstable-20.03pre204199.3140fa89c51, nixpkgs-19.09.1549.45ea6092203"`
 - channels(addy): `"home-manager, nixos-19.09.1549.45ea6092203, nixos-unstable-20.03pre204199.3140fa89c51, nixpkgs-19.09"`
 - nixpkgs: `/home/addy/.nix-defexpr/channels/nixpkgs`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: lice
# a list of nixos modules affected by the problem
module:
```

### PR DESCRIPTION
**Pull request by jorsn**
Credits to @mat8913 (https://github.com/NixOS/nixpkgs/issues/75521#issuecomment-566013855).

###### Motivation for this change

lice 0.4 is old and in nixpkgs broken: fixes #75521

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after) **+3.6 MB**
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 112895` at 9bc90032 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/60d331066bd46439d1f7be82a5987f5a">lice</a></li></li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-uri-literals.md">no-uri-literals</a>

URI literals are deprecated.
Near pkgs/tools/misc/lice/default.nix:25:16:
```
   |
25 |     homepage = https://github.com/licenses/lice;
   |                ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-explicit-check-phase.md">python-explicit-check-phase</a>

Consider using `pytestCheckHook` in `checkInputs` rather than `checkPhase = "pytest";`.

Near pkgs/tools/misc/lice/default.nix:21:3:
```
   |
21 |   checkPhase = "pytest";
   |   ^
```
</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 112895` at 9bc90032 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/f98ff2cb69fbb18d53ef6f8a922ac398">lice</a></li></li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-uri-literals.md">no-uri-literals</a>

URI literals are deprecated.
Near pkgs/tools/misc/lice/default.nix:25:16:
```
   |
25 |     homepage = https://github.com/licenses/lice;
   |                ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-explicit-check-phase.md">python-explicit-check-phase</a>

Consider using `pytestCheckHook` in `checkInputs` rather than `checkPhase = "pytest";`.

Near pkgs/tools/misc/lice/default.nix:21:3:
```
   |
21 |   checkPhase = "pytest";
   |   ^
```
</li>
  </ul>
</details>

File: pkgs/tools/misc/lice/default.nix [hunk 1]
```diff
@@ -1,21 +1,28 @@
...
 
+  propagatedBuildInputs = [
+    setuptools
+  ];
+
+  checkInputs = [
+    pytest
+  ];
+
+  checkPhase = "pytest";
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/lice/default.nix:21]: Please use pytestCheckHook.

File: pkgs/tools/misc/lice/default.nix [hunk 2]
```diff
@@ -1,21 +1,28 @@
...
+
   meta = with lib; {
-    description = "Print license based on selection and user options";
-    homepage = "https://github.com/licenses/lice";
+    description = "Print license based on selection and user options.";
+    homepage = https://github.com/licenses/lice;
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/lice/default.nix:25]:
```suggestion
    homepage = "https://github.com/licenses/lice";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/tools/misc/lice/default.nix (hunk 1 above)
Comment (jorsn) [reply pkgs/tools/misc/lice/default.nix]: fixed it. Thanks!

**Review by swflint — APPROVED** [review]
Assuming @SuperSandro2000's changes are added, LGTM

File: pkgs/tools/misc/lice/default.nix [hunk 3]
```diff
@@ -1,20 +1,22 @@
...
   };
 
+  propagatedBuildInputs = [ setuptools ];
+
+  checkInputs = [ pytestCheckHook ];
+
+
   meta = with lib; {
-    description = "Print license based on selection and user options";
+    description = "Print license based on selection and user options.";
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/lice/default.nix:19]:
```suggestion
    description = "Print license based on selection and user options";
```

Comment (jorsn) [reply pkgs/tools/misc/lice/default.nix]: done

**Review by swflint — APPROVED** [review]

File: pkgs/tools/misc/lice/default.nix [hunk 4]
```diff
@@ -1,18 +1,20 @@
-{ lib, fetchFromGitHub, python3Packages }:
+{ lib, buildPythonPackage, fetchPypi , setuptools, pytestCheckHook }:
 
-python3Packages.buildPythonPackage rec {
+buildPythonPackage rec {
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/lice/default.nix:4]:
```suggestion
```

File: pkgs/tools/misc/lice/default.nix [hunk 5]
```diff
@@ -1,18 +1,20 @@
...
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0skyyirbidknfdzdvsjga8zb4ar6xpd5ilvz11dfm2a9yxh3d59d";
   };
 
+  propagatedBuildInputs = [ setuptools ];
+
+  checkInputs = [ pytestCheckHook ];
+
+
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/lice/default.nix:17]:
```suggestion

```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Pull request merged**